assets/embauche.html

```diff
@@ -19,39 +18,38 @@
 					<option value="apprenti">apprenti.e</option>
 					<option value="stagiaire" disabled>stagiaire</option>
 					<option disabled>aide à domicile</option>
-					<option disabled>intermittent.e du spectacle</option>
+					<option disabled>intermittent.e</option>
```

Comment:
What about, rather, in CSS:
```
select[data-provides="employee"] {
	width: 6em;
}
```

![capture d ecran 2015-12-08 a 10 44 48](https://cloud.githubusercontent.com/assets/222463/11652564/d16acc16-9d98-11e5-8de5-02ac57f28020.png)
![capture d ecran 2015-12-08 a 10 44 56](https://cloud.githubusercontent.com/assets/222463/11652565/d16ad0e4-9d98-11e5-8035-d656194c85eb.png)